src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -4068,8 +4070,8 @@ public static void LastIndexOf_TurkishI_EnglishUSCulture()
         }
```

Comment:
not related to your changes: IMHO this test (`IndexOf_AllSubstrings`) is very complex and is trying to test too many test cases at the same time. It would be better to split it into few smaller tests with self-describing names

Reply:
Agreed, it's a bit of a bear. Do you have recommendations for how it might naturally be split?

Reply:
I think that the two `if` blocks could become separate tests: `value.Length == 0`, `s.Length == 0`